modules/sdk/src/main/scala/org/tessellation/sdk/package.scala

```diff
@@ -58,9 +57,6 @@ package object sdk {
       classOf[IDN] -> 528,
       classOf[ConsensusPeerDeclaration[_, _]] -> 529,
       EventTrigger.getClass -> 530,
-      TimeTrigger.getClass -> 531,
-      classOf[RegistrationExchangeRequest[_]] -> 532,
-      classOf[RegistrationExchangeResponse[_]] -> 533,
-      classOf[Deregistration[_]] -> 534
```

Comment:
it's not needed because of json-encoding in gossip, right?

Reply:
Yes, that's the reason